pkg/kube/inject/webhook.go

```diff
@@ -460,7 +461,16 @@ func addLabels(target map[string]string, added map[string]string) []rfc6902Patch
 }
 
 func updateAnnotation(target map[string]string, added map[string]string) (patch []rfc6902PatchOperation) {
-	for key, value := range added {
+	log.Errorf("howardjohn: added %v", added)
```

Comment:
remove this debugging message?

Reply:
this is his secret plan for becoming famous and getting his name printed on all the elasticsearches and splunks in the planet :P

Reply:
You caught me... just reverted